src/System.Private.CoreLib/src/System/Internal.cs

```diff
@@ -133,7 +132,6 @@ static CommonlyUsedGenericInstantiations()
             new List<ulong>();
             new List<IntPtr>();
             new List<KeyValuePair<object, object>>();
-            new List<GCHandle>();  // NCL team needs this
```

Comment:
`Dictionary<uint, WeakReference>` doesn't show up anywhere in corefx, and the only place `List<GCHandle>` is used is in `HttpListener` on Windows.